frontend/src/components/MapView/MapTooltip/index.tsx

```diff
@@ -117,6 +55,99 @@ const styles = () =>
     },
   });
 
-export interface TooltipProps extends WithStyles<typeof styles> {}
+const { multiCountry } = appConfig;
+const availableAdminLevels: AdminLevelType[] = multiCountry
+  ? [0, 1, 2]
+  : [1, 2];
+
+interface TooltipProps extends WithStyles<typeof styles> {}
+
+const MapTooltip = ({ classes }: TooltipProps) => {
+  const popup = useSelector(tooltipSelector);
+  const { i18n } = useSafeTranslation();
+  const [popupTitle, setPopupTitle] = useState<string>('');
+  const [adminLevel, setAdminLevel] = useState<AdminLevelType | undefined>(
+    undefined,
+  );
+
+  const { dataset, isLoading } = usePointDataChart();
+
+  const defaultPopupTitle = useMemo(() => {
+    if (isEnglishLanguageSelected(i18n)) {
+      return popup.locationName;
+    }
+    return popup.locationLocalName;
+  }, [i18n, popup.locationLocalName, popup.locationName]);
+
+  const renderedPopupLoader = useMemo(() => {
+    if (!popup.wmsGetFeatureInfoLoading) {
+      return null;
+    }
+    return <LinearProgress />;
+  }, [popup.wmsGetFeatureInfoLoading]);
+
+  const popupData = popup.data;
+
+  // TODO - simplify logic once we revamp admin levels ojbect
+  const adminLevelsNames = useCallback(() => {
+    const locationName = isEnglishLanguageSelected(i18n)
+      ? popup.locationName
+      : popup.locationLocalName;
+    const splitNames = locationName.split(', ');
+
+    const adminLevelLimit =
+      adminLevel === undefined
+        ? availableAdminLevels.length
+        : adminLevel + (multiCountry ? 1 : 0);
+    // If adminLevel is undefined, return the whole array
+    // eslint-disable-next-line fp/no-mutating-methods
+    return splitNames.splice(0, adminLevelLimit);
+  }, [adminLevel, i18n, popup.locationLocalName, popup.locationName]);
+
+  if (isLoading || !popup.showing || !popup.coordinates) {
+    return null;
+  }
+
+  if (dataset) {
+    return (
+      <Popup
+        coordinates={popup.coordinates}
+        className={classes.popup}
+        style={{ zIndex: 5 }}
+      >
+        <PopupPointDataChart adminLevelsNames={() => [...adminLevelsNames()]} />
+      </Popup>
+    );
+  }
+
+  return (
+    <Popup
+      coordinates={popup.coordinates}
+      className={classes.popup}
+      style={{ zIndex: 5 }}
```

Comment:
you can probably just add this to the popup class

Reply:
I didn't managed to do that. When I tried it was overided by native component class (.MapTooltip-popup-195). I tried several solution but only this one worked. If you see a way to achieved that let me know :).
![image](https://github.com/WFP-VAM/prism-app/assets/100352902/1542eaff-ef2a-457a-b668-7365616a1996)

Reply:
even with !important?